en/prepare/gnss.rst

```diff
@@ -93,6 +93,21 @@ Examples:
 I: Internal position source
 E: External (NMEA) position source
 
+Vertex log
+..........
+
+The user assigns the role "digitizing logger" on a point layer.
+
+Go to QFieldSync --> project Properties
+
+.. container:: clearer text-left
+
+  .. figure:: ../images/vertex_log.png
```

Comment:
Thanks, can you update the screenshot so it 

1. actually points to a layer
2. also includes a screenshot of the layer structure itself, ideally showing an example of a default value

@nirvn does have a sample project somewhere

Reply:
@m-kuhn is it better?